src/main/java/org/testng/remote/RemoteArgs.java

```diff
@@ -1,17 +0,0 @@
-package org.testng.remote;
-
-import com.beust.jcommander.Parameter;
-
-public class RemoteArgs {
```

Comment:
I don't see it into https://github.com/testng-team/testng-remote/tree/rm_distributed_classes/distributed/distributed/src/main/java/org/testng/remote